packages/core/src/app/checkout/getCheckoutStepStatuses.ts

```diff
@@ -90,16 +90,10 @@ const getBillingStepStatus = createSelector(
             : EMPTY_ARRAY;
     },
     (checkout, billingAddress, billingAddressFields) => {
-        const hasAddress = billingAddress
+        const isComplete = billingAddress
             ? isValidAddress(billingAddress, billingAddressFields)
             : false;
-        const isUsingWallet =
-            checkout && checkout.payments
-                ? checkout.payments.some(
-                      (payment) => SUPPORTED_METHODS.indexOf(payment.providerId) >= 0,
-                  )
-                : false;
-        const isComplete = hasAddress || isUsingWallet;
```

Comment:
Correct me if I'm wrong. As I understand from the task description, editing of billing address should be available only for PayPal. But as I see this solution affects other providers as well.